### DISCUSSION TITLE
How to load MBTiles File From Local Storage

### DISCUSSION THREAD

**Discussion by usamaahmadit8** [osmdroid/osmdroid#1968] [Q&A]
Hello all,
I am using osmdroid library with version 6.1.18 in my android java project. Is there anyone who would help me to load .mbtiles file from local storage? I want to use map offline.

**Comment by emandtf** [comment]
Why do not look in "OfflinePickerSample.java" OpenStreetMapViewer module?
The "ArchiveFileFactory" class should be able to load MBTILES file if you register its extension on it.
